### ISSUE TITLE
gnome 40 support

### ISSUE THREAD

**Issue by jmaargh** [jmaargh/titlebar-screenshot-gnome-extension#3]

**Comment by jmaargh** [comment]
Installing to a fresh Fedora 34 VM, the menu items and screenshot taking work fine, but the prefs page fails to load with the error:

```
Error: Requiring Handy, version none: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded

Stack trace:
  @/home/test/.local/share/gnome-shell/extensions/titlebar-screenshot@jmaargh.github.com/prefs.js:31:33
  _init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:206:33
  OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:122:28
  asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:115:22
  run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:177:20
  main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
  run@resource:///org/gnome/gjs/modules/script/package.js:206:19
  start@resource:///org/gnome/gjs/modules/script/package.js:190:8
  @/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```

**Comment by jmaargh** [comment]
The issue here is `const { Handy } = imports.gi;`

libhandy seems to only be available requiring Gtk3, but Gtk4 is already loaded by the host application which displays the extensions widget and both can't be loaded at the same time. There doesn't seem to be any version of libhandy for Gtk4, nor a functional libadwaita which I can use in these prefs. Ugh.

The options are therefore:
1. Redesign prefs without needing libhandy (ugly)
2. Write a clone of the widget that I'm using libhandy for (PiTA)
3. Instead of using the normal preferences window, spawn our own application for preferences (gsconnect does this) (also PiTA)

**Comment by jmaargh** [comment]
I've implemented option 3 in a5ab236
